src/core/parse.js

```diff
@@ -58,14 +58,7 @@ export let parse = (obj, selector) => {
         }
     }
 
-    // If we have properties
-    if (current[0]) {
-        // Standard rule composition
-        next = selector ? selector + '{' + current + '}' : current;
-
-        // Else just push the rule
-        return outer + next + blocks;
-    }
-
-    return outer + blocks;
+    return current[0]
+        ? outer + (selector ? selector + '{' + current + '}' : current) + blocks
+        : outer + blocks;
```

Comment:
There's already a PR tackling this if/else here https://github.com/cristianbote/goober/pull/288/files#diff-990212c4bbe286410abd4e28bab4e1c8549abc5447855b21bcdabdd65f651dd2R62

Reply:
Good catch! I went through the open PRs and missed that one.

I see that their solution slightly changes the implementation. I guess the tests are covering that correctly.